database/migrations/2021_03_29_191548_create_keys_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateKeysTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('keys', function (Blueprint $table) {
```

Comment:
Waarom noem je de tabel niet `invite-keys` en het model wel?

Reply:
Geen specifieke reden voor maar de tabel staat in de Model aangekaart dus het maakt functioneel geen verschil